packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSliderForm.component.js

```diff
@@ -72,40 +72,33 @@ const BadgeSliderForm = ({
 	const isErroneous = useMemo(() => getValidator(decimal, min, max), [decimal, min, max]);
 
 	useEffect(() => onChange(null, value), [onChange, value]);
-	const schema = {
-		autoFocus: true,
-		disabled: false,
-		type: 'number',
-		schema: {
-			minimum: min,
-			maximum: max,
-			step,
-		},
-	};
 
 	return (
 		<form className={theme('tc-badge-slider-form')} id={`${id}-slider`} onSubmit={onSubmit}>
-			<RichTooltip.RichLayout.Body
-				id={`${id}-badge-body`}
-				className={theme('tc-badge-slider-form-body')}
-			>
+			<Rich.Layout.Body id={`${id}-badge-body`} className={theme('tc-badge-slider-form-body')}>
 				<div className={theme('tc-badge-slider-form-body-row')}>
 					{icon && <Icon name={icon.name} className={theme('tc-badge-icon', icon.class)} />}
 					{editing ? (
-						<UIForm.fields.Text
+						<input
```

Comment:
UIForm fields usage here is not relevant. We want an input with the same style as UIForms, it's just an input with a classname. The Text widget is more complicated because it has descriptions and error management to display and link to the input. But here there is no need of that.

--> switch to input with the right classname.